src/LibVLCSharp.Forms/Shared/Themes/Generic.xaml

```diff
@@ -92,6 +92,7 @@
         <Setter Property="HorizontalOptions" Value="StartAndExpand" />
         <Setter Property="Margin" Value="15,15,15,0" />
         <Setter Property="FontSize" Value="Small" />
+        <Setter Property="IsVisible" Value="{TemplateBinding IsSeekBarVisible}" />
```

Comment:
This is a change in the behavior. What if people don't want to link the seekbar and the remaining/elapsed time visibility? I'd be in favor of having separate properties for that, and avoid doing that in this bugfix PR

Reply:
We discussed this internally on discord, was my idea. It does not make much sense IMHO to have the time labels (which are relative to the seekbar) without the seekbar. But no strong feelings.

Reply:
It doesn't make much sense to have a seekbar for infinite/live streams where the duration is unknown. Having the elasped time may be interesting though.
No strong feeling either, just wondering about the behavior change in a bugfix version.

Reply:
> just wondering about the behavior change in a bugfix version.

Sure :-) Happened because Laurent initially opened this PR with this screenshot and that got me wondering.

![](https://user-images.githubusercontent.com/26142591/129532254-223b68e4-c76e-4013-bfbc-e39fc3507354.png)

> for infinite/live streams where the duration is unknown. Having the elasped time may be interesting though.

Fair enough. One property for both labels though to keep it simple. We can revisit this later if needed.

Reply:
We could also hide the remaining time if the duration is unknown. The behavior is yet to be defined, and I suggest to revert this change and  take the time to discuss more about the wanted new behavior 🙂

Reply:
Sure. /cc @egbakou

Reply:
You mean hide the remaining time label if the MediaPlayer Length=0 ?

Reply:
Yes, but that's a topic for another PR I guess

Reply:
Okay thanks !
So I guess it's ok for this PR .

Reply:
@mfkl  It's done